### ISSUE TITLE
Failure in run-models workflows

### ISSUE THREAD

**Issue by joverlee521** [nextstrain/forecasts-ncov#113]
Last night's run-models workflows failed

- https://github.com/nextstrain/forecasts-ncov/actions/runs/12194752019
- https://github.com/nextstrain/forecasts-ncov/actions/runs/12192041588

```
[batch] [2024-12-06T02:53:28+00:00] Traceback (most recent call last):
[batch] [2024-12-06T02:53:28+00:00]   File "/nextstrain/build/./scripts/run-mlr-model.py", line 10, in <module>
[batch] [2024-12-06T02:53:28+00:00]     import evofr as ef
[batch] [2024-12-06T02:53:28+00:00]   File "/usr/local/lib/python3.10/site-packages/evofr/__init__.py", line 13, in <module>
[batch] [2024-12-06T02:53:28+00:00]     from .infer import InferFullRank  # , BlackJaxHandler; InferBlackJax,
[batch] [2024-12-06T02:53:28+00:00]   File "/usr/local/lib/python3.10/site-packages/evofr/infer/__init__.py", line 2, in <module>
[batch] [2024-12-06T02:53:28+00:00]     from .InferMCMC import InferMCMC, InferNUTS
[batch] [2024-12-06T02:53:28+00:00]   File "/usr/local/lib/python3.10/site-packages/evofr/infer/InferMCMC.py", line 10, in <module>
[batch] [2024-12-06T02:53:28+00:00]     from .MCMC_handler import MCMCHandler
[batch] [2024-12-06T02:53:28+00:00]   File "/usr/local/lib/python3.10/site-packages/evofr/infer/MCMC_handler.py", line 5, in <module>
[batch] [2024-12-06T02:53:28+00:00]     from jax._src.random import KeyArray
[batch] [2024-12-06T02:53:28+00:00] ImportError: cannot import name 'KeyArray' from 'jax._src.random' (/usr/local/lib/python3.10/site-packages/jax/_src/random.py)
```

**Comment by joverlee521** [comment]
The run-models workflow was successful [2 days ago](https://github.com/nextstrain/forecasts-ncov/actions/runs/12172777922). 

The new error is likely due to the [release](https://github.com/nextstrain/docker-base/actions/runs/12189741389) of a new version of docker-base yesterday.

**Comment by joverlee521** [comment]
docker-base still [installed evofr-0.1.25](https://github.com/nextstrain/docker-base/actions/runs/12189741389/job/34005493453#step:9:14963), but included [latest jax-0.4.36](https://github.com/nextstrain/docker-base/actions/runs/12189741389/job/34005493453#step:9:14966). The internal `KeyArray` alias was removed in https://github.com/jax-ml/jax/commit/fee272e550109e7409e8ae6e992bbde7bd1f1b90#diff-fc65adb03873c19c52e31d67aa2706e6eb8edeeea1f9ab5e1f6e1faf1f8a1b62. 

Will create issue in evofr.

**Comment by joverlee521** [comment]
For now I'll pin the docker image used in the run-models workflow until the fix in evofr is merged and released.

**Comment by joverlee521** [comment]
Upstream evofr issue has been resolved, waiting on release of 0.1.26 to propagate through the Nextstrain runtimes

- [ ] [docker-base](https://github.com/nextstrain/docker-base/actions/runs/12381878622)
- [ ] automated version bump PR in bioconda
- [ ] conda-base
